### ISSUE TITLE
[Legitimate Site Blocked] metadesk.run

### ISSUE THREAD

**Issue by EpicTestUser** [MetaMask/eth-phishing-detect#6608]
https://metadesk.run/

I'm using this website to do my course on Unreal Engine. It is a legitimate site providing the Unreal directly to browser. Please unblock this website,

**Comment by devenblake** [comment]
Duplicate of #6606.
